roles/splunk_common/tasks/get_facts.yml

```diff
@@ -74,28 +74,16 @@
     splunk_upgrade: "{{ 'build_location' in splunk and splunk.build_location and not splunk_install and splunk_target_version and splunk_target_version != splunk_current_version | default(False) }}"
 
 # determine if we need to set up indexer clusters
-# First, if it is manually specified in the config, use that
 - name: "Setting indexer cluster fact from config"
   set_fact:
-    splunk_indexer_cluster: "{{ splunk.indexer_cluster }}"
-  when:
-    - "'indexer_cluster' in splunk"
+    splunk_indexer_cluster: "true"
+  when: "'cluster_master_url' in splunk and splunk.cluster_master_url"
```

Comment:
Adding a note for mostly myself to revisit + maybe change this later, but I'd prefer facts like this to always be defined and have a default value. That way we don't have to rely on like "is defined" or "is not defined" to drive provisioning logic. Also this could be made into a boolean type instead of a weird string boolean.